setup.py

```diff
@@ -27,20 +27,19 @@ def find_version(*file_paths):
     packages=find_packages(),
     include_package_data=True,
     zip_safe=False,
-    install_requires=["Django>=2.2", "requests>=2.21.0"],
+    install_requires=["Django>=3.2", "requests>=2.26.0"],
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
         "Environment :: Web Environment",
         "Environment :: Web Environment :: Mozilla",
         "Framework :: Django",
         "Intended Audience :: Developers",
         "License :: OSI Approved :: BSD License",
         "Operating System :: OS Independent",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
```

Comment:
we could add `"Programming Language :: Python :: 3 :: Only"` also

Reply:
👍 I'll be updating the identifiers again after redoing the test stack to something more current anyways so I'll keep that in mind.

_(I must have followed a pattern from another moz(meao) package that also never adopted it at some point in the history…)_

BTW I'm no expert in the metadata, what's the significance of the value per se? Is it to make it explicit that it used to support v2.x but now is only v3.x compatible? Or there's another sentiment included with it over the years? Thanks!